### ISSUE TITLE
Change Project Logo

### ISSUE THREAD

**Issue by Heldaeus** [Alex313031/thorium#391]
**Is your feature request related to a problem? Please describe.**
The current logo/icon for Thorium is merely the Chromium logo with the inner ring filled with black. This by itself is not sinful but it's of my opinion that a cool project like this should have a more distinct identity.

**Describe the solution you'd like, including relevant patches or source**
I made this icon for personal use. If you'd like it to use it I can touch it up and share the source files along with any other assets you need.

![twisted-ring](https://github.com/Alex313031/thorium/assets/75914432/7dce5b7a-0e24-4bf9-a0ed-78d5db6db58a)

The element Thorium was named after Thor the Norse god. So my intention here was to play into the metallic-ness of Thor's hammer without being too on the nose. I used the twisted torus shape because it shares elements with Nordic symbology/decorations but also shares similar elements with common internet browser metaphors and the Bohr model.

If you don't like this one but are interested in changing the icon then tell me some of your ideas and I'll see what I can do.

EDIT: I changed the color grade
EDIT2: Added an explanation of the design.

**Comment by ianalexis** [comment]
I comment the same here https://github.com/Alex313031/thorium/pull/364 but closer to chromium.
I think the main devs must decide the path but the change is really needed.

**Comment by rubyFeedback** [comment]
This has been discussed before. It may be best to bundle that into a separate discussion as well as an "artists invitation" for people to design a new logo based on some guidelines what is wanted. Personally I don't have any issue with the current logo though and I also feel that people need to include the latent group of "no change is required", even though I am really mostly neutral on that topic, neither pro nor con. Just feel that those who wish for change push this more than others; I consider it such a miniscule thing to wonder about though ... :P

**Comment by rubyFeedback** [comment]
And, by the way, I like blue more than silver, for what's its worth. ;)

The thorium mineral is not the prettiest one:

https://i.imgur.com/oum5MTN.jpg

So perhaps we could compromise on silver-blueish? A bit blue to make it prettier.

**Comment by Heldaeus** [comment]
@rubyFeedback What you've linked here is Thorium ore but the element tends to get refined. So, it can look anywhere from a rusty rock, graphite-ish or dark chrome. What I've shared here is an attempt to mimic its refined state. What is above is already silver-blueish.

Here it is without the color grade:
![twisted-ring-nograde](https://github.com/Alex313031/thorium/assets/75914432/233f1b30-b2e3-4ad3-a84a-c790d7c2eab4)

Here it is with:
![twisted-ring-graded](https://github.com/Alex313031/thorium/assets/75914432/74c9c801-cdcd-4496-a1e8-af9172269892)

The element was named after Thor the Norse god. So my intention here was to play into the metallic-ness of Thor's hammer without being too on the nose. I used the twisted torus shape because it shares elements with Nordic symbology/decorations but also shares similar elements with common internet browser metaphors and the Bhor model.

So, to reiterate this is just free real-estate, I'm NOT saying that the logo should look this way but if the maintainer decides they want to use this icon I'm willing to supply what is needed. Different sizes, etc. and I can even make small symbolic versions too.
